### ISSUE TITLE
I can't install VS Code on Anaconda Navigator 

### ISSUE THREAD

**Issue by jakedoherty** [ContinuumIO/anaconda-issues#12403]
I have just purchased a Python book and the very first step is installing Anaconda Navigator and then installing and launching Microsoft Visual Studio Code through it. However, after installing Navigator there is no option to install VS Code. I have searched the internet for solutions and have found people with similar problems but I have not found a solution that works. Any help would be much appreciated I'm trying to learn Python.

**Comment by csoja** [comment]
More recent versions of Navigator do not provide the option to install VSCode through Navigator.  

You can install it and the Anaconda extension pack directly from Microsoft's page: https://code.visualstudio.com/ & https://marketplace.visualstudio.com/items?itemName=ms-python.anaconda-extension-pack 

Once you have installed it onto your machine, you can configure Navigator to point to your local VSCode installation and proceed as before. In Navigator's settings, there is a field where you can input the location of your VSCode installation if Navigator does not automatically detect it. Navigator should then display a VSCode tile on the homepage that you can launch.

**Comment by jakedoherty** [comment]
I have installed the Anaconda Extension pack (Microsoft Python extension, YAML Support by Red Hat) and I have also configured Navigator, but Navigator is still not displaying a VSCode tile on my homepage. I have attached a screenshot of my Configure Navigator prompt. Am I typing my path incorrectly?

<img width="539" alt="Screen Shot 2021-05-12 at 10 05 49 AM" src="https://user-images.githubusercontent.com/84026503/117989237-27943000-b30a-11eb-9698-c0dc340e1876.png">
